packages/dapp/src/components/OnRamper/OnRamper.tsx

```diff
@@ -45,7 +45,7 @@ export const OnRamper = () => {
     onRamperConfig.secondaryTextColor
   }&borderRadius=${onRamperConfig.borderRadius}&wgBorderRadius=${
     onRamperConfig.wgBorderRadius
-  }?apiKey=${import.meta.env.VITE_ONRAMPER_API_KEY}`;
+  }&apiKey=${import.meta.env.VITE_ONRAMPER_API_KEY}`;
```

Comment:
I would recommend using `URLSearchParams` instead. You can pass it a nice (even typed) object and let it handle the parsing into a query string.  https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#examples

Reply:
Good suggestion!

Reply:
Thanks @maxklenk! Yes, that´s cleaner and I will fix that